pkg/resources/address/address.go

```diff
@@ -101,27 +101,35 @@ func (m *ModifiersBuilder) Build(ctx context.Context) ([]func(*kubermaticv1.Clus
 		subdomain = m.seed.Spec.SeedDNSOverwrite
 	}
 
-	frontProxyLoadBalancerServiceIP := ""
+	frontProxyLBServiceIP := ""
+	frontProxyLBServiceHostname := ""
 	if m.cluster.Spec.ExposeStrategy == kubermaticv1.ExposeStrategyLoadBalancer {
 		frontProxyLoadBalancerService := &corev1.Service{}
 		nn := types.NamespacedName{Namespace: m.cluster.Status.NamespaceName, Name: resources.FrontLoadBalancerServiceName}
 		if err := m.client.Get(ctx, nn, frontProxyLoadBalancerService); err != nil {
 			return nil, fmt.Errorf("failed to get the front-loadbalancer service: %v", err)
 		}
 		// Use this as default in case the implementation doesn't populate the status
-		frontProxyLoadBalancerServiceIP = frontProxyLoadBalancerService.Spec.LoadBalancerIP
+		frontProxyLBServiceIP = frontProxyLoadBalancerService.Spec.LoadBalancerIP
 		// Supposively there is only one if not..Good luck
 		for _, ingress := range frontProxyLoadBalancerService.Status.LoadBalancer.Ingress {
 			if ingress.IP != "" {
-				frontProxyLoadBalancerServiceIP = ingress.IP
+				frontProxyLBServiceIP = ingress.IP
```

Comment:
nit: check after this loop if at least one of frontProxyLBServiceHostname,  frontProxyLBServiceIP is set.

Reply:
Here is the issue - that situation actually always happens with LoadBalancer services right after creation, before a CCM controller allocates an external IP / hostname for it. So not sure if we should produce errors in such case - maybe that was the reason for silently ignoring that here before my PR here: https://github.com/kubermatic/kubermatic/blob/9174aa67ef847947c22f4b540faebf4a6bbbfc0e/pkg/controller/seed-controller-manager/kubernetes/resources.go#L76

Reply:
OK, maybe I should do RequeueAfter instead of returning an error there - will rework it a bit.